source/nodejs/adaptivecards-designer/src/containers/viva-connections/viva-connections-container.ts

```diff
@@ -16,23 +26,26 @@ abstract class VivaConnectionsContainer extends HostContainer {
         hostElement.appendChild(vcContainer);
     }
 
+    public getHostConfig(): Adaptive.HostConfig {
+        return new Adaptive.HostConfig(
+            this._colorTheme === ColorTheme.Light ? hostConfigLight : hostConfigDark
+        );
+    }
+
+    public getBackgroundColor(): string {
+        return this._colorTheme === ColorTheme.Light ? "#F6F6F6" : "#1b1a19";
```

Comment:
pass it from constructor is better than this

Reply:
Added this with the new class